### DISCUSSION TITLE
RTL support for typography bulleted lists

### DISCUSSION THREAD

**Discussion by AldeonMoriak** [windicss/windicss#285] [Q&A]
Hello, I was wondering is there a way to change the direction of bullets in a list when the text direction is rtl?
Because when using typography plugin the bullets go to the left (default behavior) and doesn't change according to the direction of parent block.
![image](https://user-images.githubusercontent.com/37976890/118247277-3e788500-b4b8-11eb-9e22-19ff1f769c46.png)

**Comment by shiftgeist** [comment]
Tailwind suggested [tailwindcss-dir](https://github.com/RonMelkhior/tailwindcss-dir) plugin in a [discussion](https://github.com/tailwindlabs/discuss/issues/64#issuecomment-532801714) maybe something similar could be implemented as an internal plugin into windicss?
